tss-esapi/tests/common/mod.rs

```diff
@@ -172,7 +173,9 @@ pub fn get_pcr_policy_digest(
         .unwrap();
 
     if do_trial {
-        let pcr_session = context
+        // Create a trial policy session to use for calls the policy
+        // context methods.
```

Comment:
```suggestion
        // Create a trial policy session to use for calls in the policy
        // context methods.
```
?

Reply:
I think there's one more comment like this below - I'm guessing that was what you intended to write?